OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -71,9 +71,14 @@ public class MobileInfo : PausableConditionalTraitInfo, IMoveInfo, IPositionable
 		[Desc("Can move backward if possible")]
 		public readonly bool CanMoveBackward = false;
 
-		[Desc("After how many ticks the actor will turn forward during backoff")]
+		[Desc("After how many ticks the actor will turn forward during backoff.",
+		" Set it to -1 to ignore the time.")]
 		public readonly int BackwardDuration = 40;
 
+		[Desc("Actor will only try to move backwards when the path (in cells) is shorter than this value.",
+		" Set it to -1 to ignore the length of the path.")]
```

Comment:
```suggestion
		[Desc("Only try to move backwards when the path (in cells) is shorter than this value.",
		" If set to -1 backwards movement will always be allowed .")]
```

Reply:
The same there, it is better we can come up with something related to distance.

Reply:
Changed to "If set to -1 the unit will be allowed to move backwards without range limit"